### ISSUE TITLE
Two Questions: Build Steps and Bulk Plugins

### ISSUE THREAD

**Issue by redxtech** [NixNeovim/NixNeovimPlugins#113]
I'm interested in adding a couple of plugins, and have a few questions:

**First:**
I have a bunch of plugins I'd like to add. Would it be easier/better for me to add them all in a single PR? Otherwise I can continue to do a new PR per plugin.

**Second:**
how should i go about adding build steps for plugins that require them? Is that out of scope for the repo? Or should I add the build step in `overrides.nix`?

**Comment by jooooscha** [comment]
Thank you for your contributions. Always nice to see :)

For your first question, it should be fine to add all of them in one go.

For your second question: 
Could you point me to a plugin that needs any extra built steps?
I think we don't have any in the repo so I am not sure yet how to handle them

**Comment by redxtech** [comment]
Alright, I can make another PR with the rest of the plugins I want to add.

Here's a plugin that requires a build step, I've made a PR to nixpkgs to add it:

https://github.com/NixOS/nixpkgs/pull/330224/files

**Comment by jooooscha** [comment]
> I would imagine we could do the same thing here, using the overrides.nix to handle the build step, while using the current method for updating the base plugin.

That sounds good. Let me know if you need any help.

**Comment by redxtech** [comment]
It looks like you've commented out the line that includes the `overrides.nix` in the final output, is there a reason for this? Was it implemented at one point, but has since broken, so it's been commented out until it can be fixed? I tried uncommenting it, and the build broke.

For me to add the build steps, should I still work with the same `overrides.nix` file? Or should I create a new one just for build step?

If sticking with the existing one, would you be able to get what's currently there working? Or at least provide some info on what caused its removal so I can look into fixing it?

**Comment by redxtech** [comment]
It looks like you've commented out the line that includes the `overrides.nix` in the final output, is there a reason for this? Was it implemented at one point, but has since broken, so it's been commented out until it can be fixed? I tried uncommenting it, and the build broke.

For me to add the build steps, should I still work with the same `overrides.nix` file? Or should I create a new one just for build step?

If sticking with the existing one, would you be able to get what's currently there working? Or at least provide some info on what caused its removal so I can look into fixing it?

**Comment by jooooscha** [comment]
Yes, it was commented out, because I did not use it. You can just replace it with what you need and include it again.
